airflow-core/src/airflow/ui/src/pages/Pools/Pools.tsx

```diff
@@ -60,11 +71,16 @@ export const Pools = () => {
         <AddPoolButton />
       </HStack>
       <Box mt={4}>
-        {isLoading ? (
-          <Skeleton height="100px" />
-        ) : (
-          data?.pools.map((pool) => <PoolBar key={pool.name} pool={pool} />)
-        )}
+        <DataTable
+          onStateChange={setTableURLState}
+          initialState={tableURLState}  
+          isLoading={isLoading ? (
+            <Skeleton height="100px" />
+          ) : (
+            data?.pools.map((pool) => <PoolBar key={pool.name} pool={pool} />)
+          )} 
```

Comment:
```suggestion
          isLoading={isLoading}
```

We already pass the data below